### PR TITLE
Support batch segment report

### PR DESCRIPTION
**Pull request by mrproliu**
Follow the [batch segment report in main repo](https://github.com/apache/skywalking/pull/4745), make the LUA agent batch to report the segments.

### CODE REVIEW

**Comment by mrproliu** [comment]
After the [agent test tool](https://github.com/apache/skywalking-agent-test-tool/pull/15) been merged. I could adapt to the E2E test.

**Comment by mrproliu** [comment]
@wu-sheng @dmsolr It's ready for review now. :)

File: lib/skywalking/client.lua [hunk 1]
```diff
@@ -131,6 +133,36 @@ function Client:ping(metadata_buffer, backend_http_uri)
     end
 end
 
+-- Send segemnts data to backend
+local function sendSegments(segmentTransform, count, backend_http_uri)
```
Comment (wu-sheng) [inline lib/skywalking/client.lua]: Why need `count` as input and output, but do nothing?

Comment (mrproliu) [reply lib/skywalking/client.lua]: For now, yes. I will change this function return boolean type tomorrow.

**Review by wu-sheng — APPROVED** [review]
LGTM. @moonming Any suggestion?

**Review by moonming — APPROVED** [review]

**Comment by moonming** [comment]
LGTM

**Pull request merged**